util/design/lib/OtpMemMap.py

```diff
@@ -190,7 +190,8 @@ def _validate_mmap(config):
         # Place digest at the end of a partition.
         if part["sw_digest"] or part["hw_digest"]:
             if part["name"] + DIGEST_SUFFIX in item_index:
-                log.error('Digest name is not unique'.format(part["name"] + DIGEST_SUFFIX))
+                log.error('Digest name {} is not unique'.format(part["name"] +
```

Comment:
If you're rebasing anyway, I think this change should probably be in the first patch.

Reply:
right that slipped into the second commit. I'll move this hunk.